src/wp-includes/script-loader.php

```diff
@@ -1520,6 +1520,13 @@ function wp_default_scripts( $scripts ) {
 					/* translators: %s: Number of plugins. */
 					'pluginsFound'             => __( 'Number of plugins found: %d' ),
 					'noPluginsFound'           => __( 'No plugins found. Try a different search.' ),
+					'autoUpdatesEnable'        => __( 'Enable auto-updates' ),
+					'autoUpdatesEnabling'      => __( 'Enabling...' ),
+					'autoUpdatesEnabled'       => __( 'Auto-updates enabled' ),
+					'autoUpdatesDisable'       => __( 'Disable auto-updates' ),
+					'autoUpdatesDisabling'     => __( 'Disabling...' ),
+					'autoUpdatesDisabled'      => __( 'Auto-updates disabled' ),
+					'autoUpdatesError'         => __( 'The request could not be completed.' ),
```

Comment:
Consider using `wp.i18n`.

Reply:
Perhaps shiny updates should use `wp.18n`, but since this is about merging plugin/theme auto-updates into core and the new strings are "static", I don't think it would be appropriate to introduce changes to shiny updates that are not related to auto-updates.

Reply:
I don't follow, these are new strings. Existing strings should remain as is for now.